### ISSUE TITLE
URLs Impacted by Specific WCAG SC

### ISSUE THREAD

**Issue by mgifford** [GovA11y/reports#39]
Can I get a list of URLs that are tagged with axe for a specific SC? [Axe rules are tagged](https://dequeuniversity.com/rules/axe/html/4.8) with WCAG Success Criteria if they apply. Can I get a list of all URLs that fail WCAG 4.1.2 by searching for wcag412 for a specific domain?

See #27 for a comparable issue,
